inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/hive/DefaultEventFormatHandler.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.standalone.sink.hive;
+
+import org.apache.inlong.sort.standalone.channel.ProfileEvent;
+
+/**
+ * 
+ * DefaultEventFormatHandler
+ */
+public class DefaultEventFormatHandler implements IEventFormatHandler {
+
+    /**
+     * format
+     * 
+     * @param  event
+     * @param  idConfig
+     * @return
+     */
+    @Override
+    public byte[] format(ProfileEvent event, HdfsIdConfig idConfig) {
+        long msgTime = event.getRawLogTime();
+        String partitionFieldValue = idConfig.parsePartitionField(msgTime);
+        String msgTimeFieldValue = idConfig.parseMsgTimeField(msgTime);
+        byte[] partitionFieldBytes = partitionFieldValue.getBytes();
+        byte[] msgTimeFieldBytes = msgTimeFieldValue.getBytes();
+        byte[] formatBytes = new byte[partitionFieldBytes.length + 1 + msgTimeFieldBytes.length + 1
```

Comment:
ditto, it's hard to figure out the meaning of 1

Reply:
add constant variable